src/main/java/bowling/ViewUtils.java

```diff
@@ -0,0 +1,34 @@
+package bowling;
+
+import java.util.Scanner;
+
+public class ViewUtils {
+    private static final String INPUT_NULL_EMPTY_EXCEPTION = "내용을 입력해주세요.";
+    private static final String NUMBER_FORMAT_EXCEPTION = "입력된 내용이 숫자가 아닙니다.";
+    private static Scanner scanner = new Scanner(System.in);
+
+    public static void printLine(String line) {
+        System.out.println(line);
+    }
+
+    public static void print(String line) {
+        System.out.print(line);
+    }
+
+    public static String readLine() {
+        String input = scanner.nextLine();
+
+        if(input == null || input.isEmpty()) {
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
        if (input == null || input.isEmpty()) {
```

Reply:
자주 활용해야겠습니다. :)